plugins/woocommerce/client/legacy/css/twenty-twenty-two.scss

```diff
@@ -220,6 +246,24 @@ $tt2-gray: #f7f7f7;
 		padding: 1.5rem 3.5rem;
 	}
 
+	// Moved from blockthemes.css to make sure TT2 won't be changed.
+	button.button,
+	a.button {
+		background-color: var(--wp--preset--color--foreground, $primary);
+		color: var(--wp--preset--color--background, $primarytext);
+
+		&.disabled,
+		&:disabled,
+		&:disabled[disabled],
+		&.disabled:hover,
+		&:disabled:hover,
+		&:disabled[disabled]:hover {
+			background-color: var(--wp--preset--color--foreground, $primary);
+			color: var(--wp--preset--color--background, $primarytext);
+			opacity: 0.5;
```

Comment:
Nit: `opacity: 0.5` seems to be unnecessarily repeated in both files:

https://github.com/woocommerce/woocommerce/pull/35306/files#diff-4d8249aa26cad0c3201e91cebddb2c9adcf41c37c5de6f8f723b4936d2619892L56

Same for other opacities added to this file. Was it intentional?

Reply:
Not intentional, I was just being a bit lazy 😬. Removed it from TT2, since I wanted to only remove the colors from blocktheme.css. Fixed in https://github.com/woocommerce/woocommerce/pull/35306/commits/4b23164b0321515260e0ceb36af6ed1d70220430